composer.json

```diff
@@ -21,11 +21,11 @@
         "php": ">=7.2",
         "cakephp/cakephp": "^4.0",
         "cakephp/plugin-installer": "^1.0",
-        "wyrihaximus/twig-view": "dev-master"
+        "wyrihaximus/twig-view": "^5.0"
```

Comment:
Might consider `^5.0.1` here to ensure they never install the `5.0.0` version with the bug :smile: .